packages/relay/src/services/CommentService.ts

```diff
@@ -154,6 +159,31 @@ export class CommentService {
             },
         })
     }
+
+    async getCommentHistory(
+        fromEpoch: number,
+        toEpoch: number,
+        db: DB
+    ): Promise<Comment[]> {
+        if (fromEpoch > toEpoch || fromEpoch < 0 || toEpoch < 0) {
+            throw InvalidEpochRangeError
+        }
```

Comment:
```suggestion
        if (fromEpoch > toEpoch || fromEpoch < 0 || toEpoch < 0) throw InvalidEpochRangeError
```

Reply:
Prefer single line error if it's simply throwing error.

Reply:
fixed